app/app_data_dependencies.rb

```diff
@@ -0,0 +1,113 @@
+require 'yaml'
```

Comment:
Note that you've [not set your git email](https://github.com/alphagov/govuk-developer-docs/commit/1c68be66ea0062a9bcdcc4d101724fd040459e0b.patch), which is causing the commits not to be associated with your user.